lib/srv/desktop/audit_cache.go

```diff
@@ -35,169 +34,110 @@ type readRequestInfo struct {
 
 type writeRequestInfo readRequestInfo
 
-const (
-	// entryMaxItems is the maximum number of items we want
-	// to allow in a single sharedDirectoryAuditCacheEntry.
-	//
-	// It's not a precise value, just one that should give us
-	// prevent the cache from growing too large due to a
-	// misbehaving client.
-	entryMaxItems = 2000
-)
-
-type sharedDirectoryAuditCacheEntry struct {
-	nameCache         map[directoryID]directoryName
-	readRequestCache  map[completionID]readRequestInfo
-	writeRequestCache map[completionID]writeRequestInfo
-}
-
-func newSharedDirectoryAuditCacheEntry() *sharedDirectoryAuditCacheEntry {
-	return &sharedDirectoryAuditCacheEntry{
-		nameCache:         make(map[directoryID]directoryName),
-		readRequestCache:  make(map[completionID]readRequestInfo),
-		writeRequestCache: make(map[completionID]writeRequestInfo),
-	}
-}
+// maxAuditCacheItems is the maximum number of items we want
+// to allow in a single sharedDirectoryAuditCacheEntry.
+//
+// It's not a precise value, just one that should give us
+// prevent the cache from growing too large due to a
+// misbehaving client.
+const maxAuditCacheItems = 2000
 
-// totalItems returns the total numbewr of items held in the entry.
-func (e *sharedDirectoryAuditCacheEntry) totalItems() int {
+// totalItems returns the total numbewr of items held in the cache.
```

Comment:
This is only used in tests, which are single goroutine. I'm inclined to export it and make it grab the lock anyway to prevent future misuse. Thoughts?